fdbclient/SpecialKeySpace.actor.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
```

Comment:
Please add copyright information in this file as well as `fdbclient/SpecialKeySpace.actor.cpp`.

Reply:
Done. Thanks.